keps/sig-scheduling/5075-dra-consumable-capacity/kep.yaml

```
@@ -29,13 +29,13 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.34"
+latest-milestone: "v1.35"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.34"
-  beta: "v1.35"
-  stable: "v1.36"
+  alpha: "v1.35"
```

Keep the first alpha version here
```suggestion
  alpha: "v1.34"
```